modules/catalog/src/test/java/org/apache/ignite/internal/catalog/CatalogServiceSelfTest.java

```diff
@@ -1248,19 +1275,22 @@ public void testCreateIndexEvents() {
 
         // Create index.
         assertThat(service.createIndex(createIndexParams), willCompleteSuccessfully());
-        verify(eventListener).notify(any(CreateIndexEventParameters.class), ArgumentMatchers.isNull());
+        verify(eventListener, times(2)).notify(any(CreateIndexEventParameters.class), isNull());
 
         // Drop index.
         assertThat(service.dropIndex(dropIndexParams), willCompleteSuccessfully());
-        verify(eventListener).notify(any(DropIndexEventParameters.class), ArgumentMatchers.isNull());
+        verify(eventListener).notify(any(DropIndexEventParameters.class), isNull());
+
+        clearInvocations(eventListener);
 
         // Drop table.
         assertThat(service.dropTable(dropTableparams), willCompleteSuccessfully());
 
         // Try drop index once again.
         assertThat(service.dropIndex(dropIndexParams), willThrow(IndexNotFoundException.class));
 
-        verifyNoMoreInteractions(eventListener);
+        // Should have destroyed the pk index.
+        verify(eventListener).notify(any(DropIndexEventParameters.class), isNull());
```

Comment:
It is not clear, if `DropIndexEventParameters` was caused by `dropTable` or `dropIndex` call.
Let's move this higher.

Reply:
Before dropping the table, I call `clearInvocations(eventListener);`

Reply:
Revert it